docs/Beginner Tutorials/Basics.md

```diff
@@ -90,4 +90,4 @@ While building, drones will stop assisting and quickly mine ores that are runnin
 - You can confirm a block, cancel the build (this will stop your mech from building, but the confirmed build will remain) and continue this one block a time to set up a large queue. This can be helpful when you want to get something started, but need to address something else first. Drones will not work on these queues, so this is only a placeholder.
 - Containers making contact with your Core will act like extra inventory for your Core. This not only gives more inventory space, but allows for more space to input raw materials, as well as output raw materials from your inventory (See Unloader) An Unloader will take materials out of both the Core or connected Containers because they now share the same inventory space.
 
-<img src="https://i.imgur.com/vcXy8EZ.png" width=40%>
\ No newline at end of file
+<img src="https://i.imgur.com/vcXy8EZ.png" width=40%>
```

Comment:
This was already part of the old page, but I don't think it's a good idea to ever have imgur links here-- they can break after a while, as was demonstrated with the Mindustry README. Instead, images should be placed on the wiki itself and linked directly.

Reply:
True! ![](https://preview.redd.it/0ppv89s4exj21.jpg?width=640&auto=webp&s=3adf6a6f4e6d0c748f7e353300ea43e156b75a14)